include/string.h

```diff
@@ -52,6 +52,7 @@ char* __fastcall__ strchr (const char* s, int c);
 int __fastcall__ strcmp (const char* s1, const char* s2);
 int __fastcall__ strcoll (const char* s1, const char* s2);
 char* __fastcall__ strcpy (char* dest, const char* src);
+char* __fastcall__ stpcpy (char* dest, const char* src);
```

Comment:
The prototype should be moved below to the non standard ones

Reply:
Oops! True. Sorted.